backend/handlers/invite_handler.py

```diff
@@ -70,6 +71,11 @@ def patch(self, user, invite_key):
         data = self.request.body
 
         invite = ndb.Key(urlsafe=invite_key).get()
+
+        Utils._assert(invite.status == 'accepted', 
```

Comment:
SE certificar se esse é o handler para aceitar convite quando convida um membro e quando convida uma instituição. Acredito que outros Handerls mudam o status de um invite de sent to accepted.

Reply:
Done